app/grandchallenge/reader_studies/views.py

```diff
@@ -395,7 +396,7 @@ class ReaderStudyDisplaySetList(CivSetListView):
         *CivSetListView.columns,
     ]
 
-    default_sort_column = 3
+    default_sort_column = 2
```

Comment:
Why this change? Now it will be ordered by pk. I think we should keep the default ordering by order.

Reply:
You are right, I neglected that display sets have the addition of order in the mix. It was originally meant to have it equal between Archives and Display Sets. Reverted!